lib/AliasFieldPlugin.js

```diff
@@ -41,11 +40,11 @@ AliasFieldPlugin.prototype.apply = function(resolver) {
 			path: request.descriptionFileRoot,
 			request: data
 		});
-		resolver.doResolve(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", createInnerCallback(function(err, result) {
+		resolver.doResolveWithErrorLogging(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", function(err, result) {
```

Comment:
This is the control flow when using `doResolveWithErrorLogging` inside of plugins:

```
resolve -> doResolveWithErrorLogging -> SomePlugin -> doResolveWithErrorLogging -> SomeOtherPlugin -> error
                                                                          v-----------------------------+
                                                                     (with log) -> SomeOtherPlugin -> error
                               v------------------------------------------*-----------------------------+
                          (with log) -> SomePlugin -> doResolveWithErrorLogging -> SomeOtherPlugin -> error
                                                                          v-----------------------------+
                                                                     (with log) -> SomeOtherPlugin -> error
   v---------------------------*------------------------------------------*-----------------------------+
```

This is the control flow when using doResolve:

```
resolve -> doResolveWithErrorLogging -> SomePlugin -> doResolve -> SomeOtherPlugin -> error
                               v--------------------------------------------------------+
                          (with log) -> SomePlugin -> doResolve -> SomeOtherPlugin -> error
   v---------------------------*--------------------------------------------------------+
```